### DISCUSSION TITLE
Custom back button  in dialog

### DISCUSSION THREAD

**Discussion by sasikr** [serenity-is/Serenity#6054] [Help]
how to customize back button in dialog, custom back button url
![image](https://user-images.githubusercontent.com/62542404/138642865-b0f759e2-e548-4b5b-ad37-d71a69eb669c.png)

**Comment by minhhungit** [comment]
panel dialog is also a dialog, so, to implement back button, you just need to call function ex `this.dialogClose()`

**Reply by minhhungit** [reply]
Or you can redirect to new page using code
https://github.com/serenity-is/Serenity/issues/5081#issuecomment-650282960
